### ISSUE TITLE
Non defined annotation "DymolaStoredErrors" present in MSL

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#192]
**Modified by dietmarw on 15 Jul 2009 09:32 UTC**
While working on #191 I noticed a non standard annotation "`DymlaStoredErrors" being present in 
```
Analog
  Basic.mo
     1930: DymolaStoredErrors); 
  Interfaces.mo
      347: DymolaStoredErrors, 
  Semiconductors.mo
     1814: DymolaStoredErrors);
```
As each of the annotations is empty they should just be removed. (Going to do this later today.)

----

**Reported by dietmarw on 15 Jul 2009 09:32 UTC**
While working on #191 I noticed a non standard annotation "`DymlaStoredErrors" being present in 
```
Analog
  Basic.mo
     1930: DymolaStoredErrors); 
  Interfaces.mo
      347: DymolaStoredErrors, 
Semiconductors.mo
     1814: DymolaStoredErrors);
```
As each of the annotations is empty they should just be removed. (Going to do this later today.)

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/192

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 15 Jul 2009 09:32 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 15 Jul 2009 12:37 UTC**
fixed in aa9133113e4848728acd80f46d858a5017754b78
